packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -133,6 +141,102 @@ export class ApexDebug extends DebugSession {
     this.sendResponse(response);
   }
 
+  protected async setBreakPointsRequest(
+    response: DebugProtocol.SetBreakpointsResponse,
+    args: DebugProtocol.SetBreakpointsArguments
+  ): Promise<void> {
+    if (args.source && args.source.path) {
+      const processedBreakpoints: DebugProtocol.Breakpoint[] = [];
+      const uriPath = this.convertClientPathToDebugger(args.source.path);
+
+      try {
+        const linesToSetBreakpoint = await this.myBreakpointService.reconcileBreakpoints(
+          this.sfdxProject,
+          this.mySessionService.getSessionId(),
+          uriPath,
+          args.lines
+        );
+
+        for (const clientLine of linesToSetBreakpoint) {
+          const serverLine = this.convertClientLineToDebugger(clientLine);
+          const typeref = this.myBreakpointService.getTyperefFor(
+            uriPath,
+            serverLine
+          );
+          if (typeref && typeref.length > 0) {
+            const cmdOutput = await this.myBreakpointService.createLineBreakpoint(
+              this.sfdxProject,
+              this.mySessionService.getSessionId(),
+              typeref,
+              serverLine
+            );
+            this.myBreakpointService.cacheLiveBreakpoint(
+              uriPath,
+              clientLine,
+              cmdOutput.getId()
+            );
+            processedBreakpoints.push(
+              <DebugProtocol.Breakpoint>{
```

Comment:
I think the recommended way to cast is https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#new-tsx-file-extension-and-as-operator 

So use `as DebugProtocol.Breakpoint` instead of <DebugProtocol.Breakpoint>.